### PR TITLE
Add large models and modify download link of resnet34

### PR DESCRIPTION
**Pull request by xyn1201**
Hello, your work helps me a lot while doing my research!

To make this code easier to use, I made these changes:
1. Add Large scale models, TransFuse_L, TransFuse_L_384, which were mentioned in the paper.
2. Modify the download link of resnet34 in README, because the link writen in lib/TransFuse.py was different from the one in README.
3. Add `align_corners=True` to `F.interpolate`, in order to avoid such WARNING
```
/usr/local/miniconda3/lib/python3.7/site-packages/torch/nn/functional.py:3455: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode)
```

### CODE REVIEW

**Comment by AasiaRehman** [comment]
how to download pretrained models deit_small_distilled_patch16_224

**Comment by xyn1201** [comment]
> how to download pretrained models deit_small_distilled_patch16_224

I didn't use 'deit_small_distilled_patch16_224'

In lib/DeiT.py, I just used the 'deit_tiny_patch16_224', 'deit_small_patch16_224' and 'deit_base_patch16_224'.

Is there anything wrong with it?

**Comment by AliMedVisionTech** [comment]
Hi how to generate the final mask of the model via test.py file ?

**Comment by xyn1201** [comment]
> Hi how to generate the final mask of the model via test.py file ?

Sorry, I did not provide the pre-trained TransFuse_L or TransFuse_L_384,
because I only use these models for my own research, 
my pre-trained models are trained with some real-world private data, not public datasets.


But if you mean, how to generate the final mask of the model TransFuse_S, please see
https://github.com/Rayicer/TransFuse/blob/main/README.md#experiments
and refer to 2. Testing

https://github.com/Rayicer/TransFuse/blob/3f50ab61cab7931ef8aefd6c9c0b2ba9b031ff76/test_isic.py#L76
res represents the final mask, and here saves res to disk.
